compiler/test/dotty/tools/backend/jvm/DottyBytecodeTests.scala

```diff
@@ -401,55 +401,6 @@ class TestBCode extends DottyBytecodeTest {
     }
   }
 
-  @Test def returnThrowInPatternMatch = {
-    val source =
-      """class Test {
-        |  def test(a: Any): Int = {
-        |    a match {
-        |      case _: Test => ???
-        |    }
-        |  }
-        |}
-      """.stripMargin
-
-    checkBCode(source) { dir =>
-      val moduleIn = dir.lookupName("Test.class", directory = false)
-      val moduleNode = loadClassNode(moduleIn.input)
-      val method = getMethod(moduleNode, "test")
-
-      val instructions = instructionsFromMethod(method)
```

Comment:
I agree this test is not maintainable. It tests that there is no `return` generated by pattern matcher and could be much simpler. I suggest rewriting the test as follow:
```scala
...
val instructions = instructionsFromMethod(method)
val hasReturn = instruction.exists {
  case Op(Opcodes.RETURN)) => true
  case _ => false
}
assertFalse(hasReturn)
```

Reply:
That looks better, yes!